sidebars.js

```diff
@@ -43,23 +43,17 @@ module.exports = {
       type: 'category',
       label: 'Core concepts',
       items: [
-        'core/using-okteto-cli',
-        {
-          type: 'category',
-          label: 'Okteto components',
-          items: [
-            'core/components/development-containers',
```

Comment:
I think you need to add a redirect for this one of the URL okteto.com/docs/core/components/namespace will break

Reply:
https://deploy-preview-641--okteto-docs.netlify.app/docs/1.18/core/components/namespace results in a 404

Reply:
It's `namespaces`, not `namespace`. 
This PR doesn't change URLs, it doesn't require any redirects. We don't want to add any redirect at all until the docs revamp is finished. After that, we will probably have a single PR with redirects.
The only redirect is to remove `/docs/guides/tutorials/getting-started-with-pipelines` in favor of `/docs/guides/tutorials/getting-started-with-okteto`